.circleci/config.yml

```diff
@@ -0,0 +1,238 @@
+version: 2.1
+
+orbs:
+  browser-tools: circleci/browser-tools@1.1.1
+
+skip_pr_branches: &skip_pr_branches
+  filters:
+    branches:
+      ignore: /^pull\/.*$/
+
+executors:
+  amphtml-executor:
+    machine:
+      image: ubuntu-2004:202010-01
+    resource_class: xlarge
+
+commands:
+  setup_vm:
+    steps:
+      - checkout
+      - browser-tools/install-chrome:
+          replace-existing: true
+      - run:
+          name: 'Install Google Cloud SDK'
+          command: |
+            cd ~/
+            curl -Ss --retry 5 https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-323.0.0-linux-x86_64.tar.gz | tar xz
+            echo 'source ~/google-cloud-sdk/path.bash.inc' >> $BASH_ENV
+      - run:
+          # TODO(rsimha, ampproject/amp-github-apps#1110): Update storage details.
+          name: 'Setup Storage'
+          command: |
+            openssl aes-256-cbc -k $GCP_TOKEN -in build-system/common/sa-travis-key.json.enc -out sa-travis-key.json -d
+            gcloud auth activate-service-account --key-file=sa-travis-key.json
+            gcloud config set account sa-travis@amp-travis-build-storage.iam.gserviceaccount.com
+            gcloud config set pass_credentials_to_gsutil true
+            gcloud config set project amp-travis-build-storage
+      - run:
+          name: 'Configure Hosts'
+          command: |
+            echo "127.0.0.1 ads.localhost" | sudo tee -a /etc/hosts
+            echo "127.0.0.1 iframe.localhost" | sudo tee -a /etc/hosts
+            echo "127.0.0.1 jgla3zmib2ggq5buc4hwi5taloh6jlvzukddfr4zltz3vay5s5rq.recaptcha.localhost" | sudo tee -a /etc/hosts
+            echo "127.0.0.1 fonts.googleapis.com" | sudo tee -a /etc/hosts
```

Comment:
nit: curious if there's a way to save these in a file and just `cat` the file into /etc/hosts

Reply:
Good idea. Will attempt this via a separate PR.